src/messaging/create.js

```diff
@@ -53,22 +53,24 @@ module.exports = function (Messaging) {
 		var mid;
 		var message;
 		var isNewSet;
+		const timestamp = data.timestamp || new Date().getTime();
+		if (!callback) {
+			callback = function () {};
+		}
 
 		async.waterfall([
-			function (next) {
-				Messaging.checkContent(data.content, next);
```

Comment:
Why is this removed? Maybe wrap it in `if (!data.system)` if you don't want system messages to be validated.

Reply:
Was initially removed because `.sendMessage()` contains the same call to check content. However, a data integrity check inside of `.addMessage()` is okay as well, as it might be called by plugins, etc.

I'll add it back in